akka-docs/src/main/paradox/discovery/index.md

```diff
@@ -206,8 +206,7 @@ Each service can have multiple endpoints.
 
 ## Discovery Method: Aggregate multiple discovery methods
 
-Aggregate discovery allows multiple discovery methods to be aggregated e.g. try and resolve
-via DNS and fall back to configuration.
+Aggregate discovery allows multiple discovery methods to be aggregated.
```

Comment:
I dropped the 'e.g.' part. Looks strange to start talking about it on the first sentence. Later, we give an example already and explain how the ordering works.